.github/workflows/unit-test.yml

```diff
@@ -64,6 +64,7 @@ jobs:
     runs-on: windows-latest
     env:
       NPM_CONFIG_UNSAFE_PERM: true
+      NODE_OPTIONS: --max-old-space-size=4096
```

Comment:
not sure if its worth to check that the fix doesnt create any leak that for some reasons is only a problem in windows ?

Reply:
https://github.com/open-telemetry/opentelemetry-js/issues/3628